UI/js/utils/utils.js

```diff
@@ -98,12 +98,13 @@ $(document).ready(function(){
       // pick a random image and assign it to the body element
       var ramdomImage = _.sample(data);
 
-      //need to get absolute path to deal with Chrome/Safari
-      document.body.style.backgroundImage = 'url(' + window.location + 'external/image/' + ramdomImage + ')';
+      document.body.style.backgroundImage = 'url(external/image/' + ramdomImage + ')';
+      document.body.style.backgroundColor = '#444444';
     })
     .fail(function() {
       // select a default image and assign it to the body element
-      document.body.style.backgroundImage = 'url(' + window.location + 'data/images/back/default.jpg' + ')';
+      document.body.style.backgroundImage = 'url(data/images/back/default.jpg)';
+      document.body.style.backgroundColor = '#444444';
```

Comment:
It's a small thing, but why set the BG color in two places?  Couldn't it just come after `fail()` and it will be executed whether or not there was a failure?  If not, doesn't jQuery's `ajax()` support something like `finally` along with `done()` and `fail()`?  If so, that would be a better place for it.